api-reference/20 Data Visualization Widgets/dxPieChart/1 Configuration/annotations/annotations.md

```diff
@@ -5,7 +5,207 @@ inherits: dxPieChart.Options.commonAnnotationSettings
+        styleUrls: ['./app.component.css']
+    })
+    export class AppComponent {
+        // ...
+    }
+
+    <!-- tab: app.module.ts -->
+    import { BrowserModule } from '@angular/platform-browser';
+    import { NgModule } from '@angular/core';
+    import { AppComponent } from './app.component';
+
+    import { DxPieChartModule } from 'devextreme-angular';
+
+    @NgModule({
+        declarations: [
+            AppComponent
+        ],
+        imports: [
+            BrowserModule,
+            DxPieChartModule
+        ],
+        providers: [ ],
+        bootstrap: [AppComponent]
+    })
+    export class AppModule { }
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxPieChart ... >
+            <DxAnnotation
+                type="text"
+                text="Annotation text"
+            />
+            <DxAnnotation
+                type="image"
+                image="http://image/url/myimage.png"
+            />
+            <DxAnnotation
+                type="custom"
+                template="custom-annotation"
+            />
+            <template #custom-annotation="{ data }">
+                <svg>
+                    <!-- Declare custom SVG markup here -->
+                </svg>
+            </template>
+        </DxPieChart>
+    </template>
+
+    <script>
+    import DxPieChart, {
+        DxAnnotation
+    } from 'devextreme-vue/pie-chart';
+
+    export default {
+        components: {
+            DxPieChart,
+            DxAnnotation
+        },
+        data() {
+            // ...
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+
+    import PieChart, {
+        Annotation
+    } from 'devextreme-react/pie-chart';
+
+    function CustomAnnotation(annotation) {
+        const data = annotation.data;
+        return (
+            <svg>
+                {/* Declare custom SVG markup here */}
+            </svg>
+        );
+    }
+
+    export default function App() {
+        return (
+            <PieChart ... >
+                <Annotation
+                    type="text"
+                    text="Annotation text"
+                />
+                <Annotation
+                    type="image"
+                    image="http://image/url/myimage.png"
+                />
+                <Annotation
+                    type="custom"
+                    render={CustomAnnotation}
+                />
+            </PieChart>
+        );
+    }
+
+---
+
+Annotations can be anchored to a **PieChart** element. To do this, use the [argument]({basewidgetpath}/Configuration/annotations/#argument) or [series]({basewidgetpath}/Configuration/annotations/#series) options, depending on the element you want to anchor the annotation to.
+
+To define the position of an unanchored annotation, set the pixel coordinates ([x]({basewidgetpath}/Configuration/annotations/#x) and [y]({basewidgetpath}/Configuration/annotations/#y)) options.
```

Comment:
The list with examples is detached from the paragraph about anchored elements, although it illustrates how to position them. I suggest taking the text from the [Chart description](http://jsserver.corp.devexpress.com:8080/Documentation/20_2/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/):

```suggestion
Annotations can be unattached or anchored to a chart element. The following list shows how to position them. Chart coordinates ([argument]({basewidgetpath}/Configuration/annotations/#argument) and [series]({basewidgetpath}/Configuration/annotations/#series)) specify the element that the annotation's arrow points to; pixel coordinates ([x]({basewidgetpath}/Configuration/annotations/#x) and [y]({basewidgetpath}/Configuration/annotations/#y)) specify the position of the annotation's center.
```

Reply:
BTW, the PolarChart's annotations [have the same issue](http://jsserver.corp.devexpress.com:8080/Documentation/20_2/ApiReference/Data_Visualization_Widgets/dxPolarChart/Configuration/annotations/). Let's fix it there too.